code/modules/mob/living/simple_animal/hostile/megafauna/wendigo.dm

```diff
@@ -283,16 +283,45 @@ Difficulty: Hard
 	exit.set_light(20, 1, COLOR_SOFT_RED)
 	return ..()
 
-/obj/projectile/colossus/wendigo_shockwave
+/obj/projectile/wendigo_shockwave
 	name = "wendigo shockwave"
+	icon_state = "chronobolt"
+	damage = 25
+	armour_penetration = 100
+	speed = 2
+	damage_type = BRUTE
+	pass_flags = PASSTABLE
 	/// If wave movement is enabled
 	var/wave_movement = FALSE
 	/// Amount the angle changes every pixel move
 	var/wave_speed = 15
 	/// Amount of movements this projectile has made
 	var/pixel_moves = 0
 
-/obj/projectile/colossus/wendigo_shockwave/pixel_move(trajectory_multiplier, hitscanning = FALSE)
+/obj/projectile/wendigo_shockwave/can_hit_target(atom/target, direct_target = FALSE, ignore_loc = FALSE, cross_failed = FALSE)
+	if(isliving(target))
+		direct_target = TRUE
+	return ..(target, direct_target, ignore_loc, cross_failed)
+
+/obj/projectile/wendigo_shockwave/on_hit(atom/target, blocked = 0, pierce_hit)
+	. = ..()
+	if(isliving(target))
```

Comment:
ditto on adding early returns... also couldn't you make a common proc on `/obj/projectile` or something where you could just invoke it and listen for the result?

Reply:
just anything to cut down on the copypaste really because stuff like this tends to desync from each-other fast